homeassistant/components/mcp_server/strings.json

```diff
@@ -0,0 +1,18 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "The Model Context Protocol (MCP) Server  integration exposes the Home Assistant Assist API to LLM clients on your network. MCP is an open protocol that standardizes how applications provide context to Large Language Models.\n\nThe Home Assistant MCP server requires authentication, such as a long lived access token. See the integration documentation for [more info]({more_info_url}).",
```

Comment:
I think that we don't have to explain what the integration does. That's for the docs to solve.

It's also not just Assist API. It's any LLM API. It allows external Large Language Model to interact with Home Assistant's internal LLM tools.